config/initializers/redis.rb

```diff
@@ -1,11 +1,11 @@
-REDIS_POOL = ConnectionPool.new(size: 10) do
+REDIS_POOL = ConnectionPool.new(size: IdentityConfig.store.redis_pool_size) do
   Redis::Namespace.new(
     'redis-pool',
     redis: Redis.new(url: IdentityConfig.store.redis_url),
   )
 end
 
-REDIS_THROTTLE_POOL = ConnectionPool.new(size: 5) do
+REDIS_THROTTLE_POOL = ConnectionPool.new(size: IdentityConfig.store.redis_throttle_pool_size) do
```

Comment:
looks like we initialize the attempts API redis outside of this file:

https://github.com/18F/identity-idp/blob/34f0bccd457e3ee7fe103f7e073d5fa278e9127c/app/services/irs_attempts_api/redis_client.rb#L3-L9

Maybe if not for this PR, but could we follow up and add a config for it too, maybe even move it inside this file as a constant instead of a class member there?